00-Introducing-HOOMD-blue/03-Initializing-the-System-State.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<p>We are also purposely starting in a low density system to enable quick randomization and compression. It might be helpful to mention this is a good paradigm to use for generating initial conditions. It is the model you show, and you mention the low density in the next notebook, but just to be more explicit about it here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>done.</p>